lumberjack_test.go

```diff
@@ -127,6 +127,36 @@ func TestDefaultFilename(t *testing.T) {
 	existsWithContent(filename, b, t)
 }
 
+func TestRollingInterval(t *testing.T) {
+	currentTime = fakeTime
+	bCount := 0
+	dir := makeTempDir("RollingInterval", t)
+	defer os.RemoveAll(dir)
+	filename := logFile(dir)
+	l := &Logger{
+		Filename:        filename,
+		MaxSize:         100,
+		RollingInterval: 2,
+	}
+	defer l.Close()
+	b := []byte("boo!")
+	allBytes := append(b, b...)
+	for i := 0; i < 2; i++ {
+		n, err := l.Write(b)
+		isNil(err, t)
+		bCount += n
+	}
+	equals(len(allBytes), bCount, t)
+	existsWithContent(filename, allBytes, t)
+	fileCount(dir, 1, t)
+	<-time.After(time.Millisecond * 2000)
```

Comment:
Not a big fan of sleeping in tests. Set the time into the past so it gets rotated right away.